projects/components/src/table/table.component.ts

```diff
@@ -73,7 +73,7 @@ import {
               [columnConfig]="columnDef"
               [index]="index"
               [sort]="columnDef.sort"
-              [values]="this.getAvailableValues(columnDef) | async"
+              [values]="this.getAvailableValues(columnDef)"
```

Comment:
This still has the same issue as before - it's returning a new reference on each invocation